### DISCUSSION TITLE
how to run the install file "deepmd-kit-2.0.0.b3-cuda11.1_gpu-Linux-x86_64.sh.1"

### DISCUSSION THREAD

**Discussion by tonystarkiss** [deepmodeling/deepmd-kit#897] [Q&A]
root@f9730af6cc03:/data# ./deepmd-kit-2.0.0.b3-cuda11.1_gpu-Linux-x86_64.sh.1 
./deepmd-kit-2.0.0.b3-cuda11.1_gpu-Linux-x86_64.sh.1: line 1: $'\367#\377\305I\337\374\2665\323\031}b\fm\251A\016\026\235\353\256e\342\320': command not found
./deepmd-kit-2.0.0.b3-cuda11.1_gpu-Linux-x86_64.sh.1: line 2: syntax error near unexpected token `)'
./deepmd-kit-2.0.0.b3-cuda11.1_gpu-Linux-x86_64.sh.1: line 2: `얻ϯ䅠ɴ\_3׺¶q^Y§³*!I%g+꦳f]Y0\&¤ﾼ3Gj/Jʃψ烏¬ذۉ)l¡L¾H½1x¥58oh¤ 긩'

**Comment by njzjz** [comment]
See #740.

**Reply by tonystarkiss** [reply]
thanks, why not provide a merged install file?

**Reply by njzjz** [reply]
GitHub has a limitation on the size of a single file (2GB).
